tests/python/test_views.py

```diff
@@ -35,11 +38,14 @@ def setUp(self):
             self.run = dict(res.fetchone())
 
     def tearDown(self):
-        with tables(db.engine, 'projects', 'bams', 'vcfs') as (con, projects, bams, runs):
+        with tables(db.engine, 'projects', 'bams', 'vcfs', 'user_comments') \
```

Comment:
Weird, I thought I posted this comment on another PR but maybe not: anyway, maybe turn this `delete_tables(comments, runs, bams, projects)` into a helper?